lib/rules/dashed-ident-no-missing-reference-function/index.js

```diff
@@ -0,0 +1,61 @@
+// @ts-nocheck
+
+'use strict';
+
+const valueParser = require('postcss-value-parser');
+
+const declarationValueIndex = require('../../utils/declarationValueIndex');
+const report = require('../../utils/report');
+const ruleMessages = require('../../utils/ruleMessages');
+const validateOptions = require('../../utils/validateOptions');
+
+const ruleName = 'dashed-ident-no-missing-reference-function';
+
+const messages = ruleMessages(ruleName, {
+	rejected: (dashedIdent) =>
+		`Unexpected missing dashed-ident reference function for "${dashedIdent}"`,
+});
+
+const REFERENCE_FUNCTIONS = ['color', 'env', 'var'];
```

Comment:
Curious - are there many other uses cases for this set of reference functions? Can't think of any in my normal use-cases, but thinking about if we should pull this out to the constants files.

Reply:
I think we can pull them out into a reference file if they end up being shared by another rule.